htlcswitch/switch.go

```diff
@@ -36,6 +37,10 @@ const (
 	// DefaultAckInterval is the duration between attempts to ack any settle
 	// fails in a forwarding package.
 	DefaultAckInterval = 15 * time.Second
+
+	// DefaultHTLCExpiry is the duration after which Adds will be cancelled
+	// if they could not get added to an outgoing commitment.
+	DefaultHTLCExpiry = time.Minute
```

Comment:
At a glance, my intuition tells me this value can likely be much shorter, but I don't see much harm with starting out with a bit more slack, then tightening it in the future once we can observe the behavior of this change a bit more in the wild (should make sure we add adequate logging to so we can easily trace instances of us cancelling back undelivered HTLCs).

Reply:
yeah i left the longer timeout to handle restarts, since we do the staggered reconnections so I wanted a little bit of time for things to shake out before failing. it probably could be lower in the steady state, but using the same failure throughout simplifies the implementation for now (since the expiries in the queue are guaranteed to be in order)